expansion/src/main/java/no/unit/nva/expansion/model/ExpandedMessage.java

```diff
@@ -27,6 +27,7 @@
     private MessageStatus status;
     private String sender;
     private SortableIdentifier resourceIdentifier;
+    private URI resourceId;
```

Comment:
Should these not all be "id"?

Reply:
I think that the misunderstanding is actually the term Resource. We use it in two different ways. One way is as a better name to Publication, but we also view everything in the database as a Resource. e.g. a message is a Resource and a DoiRequest is a Resource. We can agree that all things are Resources, but we should find a name for the Publication that is unique.

Reply:
Skal vi ta det i https://unit.atlassian.net/browse/NP-3524 ?